lib/interceptor.js

```diff
@@ -24,7 +24,8 @@ function Interceptor(scope, uri, method, requestBody, interceptorOptions) {
     this.interceptorMatchHeaders = [];
     this.method = method.toUpperCase();
     this.uri = uri;
-    this._key = this.method + ' ' + scope.basePath + scope.basePathname + (typeof uri === 'string' ? '' : '/') + uri;
+    this._key = this.method + ' ' + scope.basePath + scope.basePathname + (typeof uri === 'string' ? '' : '/') +
```

Comment:
The new logic for stringifying the function means filters will produce identical keys, where they did not previously. This may break existing behavior. PR will not be accepted with this change, please remove.